guides/common/modules/proc_uploading-additional-scap-content.adoc

```diff
@@ -0,0 +1,39 @@
+[id="Uploading_Additional_SCAP_Content_{context}"]
+= Uploading Additional SCAP Content
+
+You can upload additional SCAP content into {ProjectServer}, either content created by yourself or obtained elsewhere.
+For example, you can get the latest OpenSCAP contents for additional systems from the https://github.com/ComplianceAsCode/content/releases[SSG GitHub repository].
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-Uploading_Additional_SCAP_Content_{context}[CLI procedure].
+
+.Prerequisite
+* Your user account has the `create_scap_contents` permission.
+* You have downloaded a SCAP content archive file.
+
+.Procedure
+. Unpack the SCAP content archive file on your local machine.
+. In the {ProjectWebUI}, navigate to *Hosts* > *SCAP contents*.
+. Click *Upload New SCAP Content*.
+. Enter a title in the *Title* text box, such as `_My SCAP Content_`.
+. In *Scap File*, click *Choose file*, navigate to the location containing a SCAP data-stream file and click *Open*.
+. On the *Locations* tab, select locations.
+. On the *Organizations* tab, select organizations.
+. Click *Submit*.
+
+If the SCAP content file is loaded successfully, a message similar to `Successfully created _My SCAP Content_` is displayed.
+
+[id="cli-Uploading_Additional_SCAP_Content_{context}"]
+.CLI procedure
+. Unpack the SCAP content archive to a directory on your {ProjectServer}, such as `_/usr/share/xml/scap/my_content/_`.
+. Run the following Hammer command on {ProjectServer}:
++
+[options="nowrap", subs="+quotes,attributes,verbatim"]
+----
+# hammer scap-content bulk-upload --type directory \
```

Comment:
Could not reproduce this command on Foreman 3.5: 

````
# hammer scap-content bulk-upload --type directory --directory _/usr/share/xml/scap/my_content/_ --location "Example" --organization "Munich"
Error: No such sub-command 'scap-content'.

See: 'hammer --help'.
````

Reply:
If you couldn't enable Openscap in Hammer CLI, then it's logical that you couldn't run this command as a consequence.
Otherwise, it has been in Hammer much longer: https://docs.theforeman.org/3.1/Administering_Project/index-katello.html#Uploading_Extra_SCAP_Content_admin

Reply:
Works!